HISTORY.rst

```diff
@@ -3,6 +3,100 @@
 History
 =======
 
+1.1.1 (2018-07-18)
+------------------
+
+This is a bugfix-only release for Python 2 and migration issues found
+in dj-stripe 1.1.0.
+
+If you are on an older version of dj-stripe, you should upgrade to 1.1.x
+first with `pip install "djstripe<1.2"` before upgrading to 1.2.x.
+
+Please make sure to read the 1.1.0 release notes below.
+
+
+1.1.0 (2018-06-11)
+------------------
+
+In dj-stripe 1.1.0, we made a *lot* of changes to models in order to
+bring the dj-stripe model state much closer to the upstream API objects.
+If you are a current user of dj-stripe, you will most likely have to
+make changes in order to upgrade. Please read the full changelog below.
+If you are having trouble upgrading, you may ask for help `by filing an
+issue on GitHub`_.
+
+Upgrade notes
+-------------
+
+Migration reset
+~~~~~~~~~~~~~~~
+
+The next version of dj-stripe, **1.2.0**, will reset all the migrations
+to ``0001_initial``. Migrations are currently in an unmaintainable
+state.
+
+**What this means is you will not be able to upgrade directly to
+dj-stripe 1.2.0. You must go through 1.1.0 first, run
+``manage.py migrate djstripe``, then upgrade to 1.2.0.**
+
+Python 2.7 end-of-life
+~~~~~~~~~~~~~~~~~~~~~~
+
+dj-stripe 1.1.0 drops support for Django 1.10 and adds support for
+Django 2.0. Django 1.11+ and Python 2.7+ or 3.4+ are required.
+
+Support for Python versions older than 3.5, and Django versions older
+than 2.0, will be dropped in dj-stripe 1.3.0.
+
+Backwards-incompatible changes and deprecations
+-----------------------------------------------
+
+Removal of polymorphic models
```

Comment:
Flagging `Backwards-incompatible` changes and deprecations for folks to easily spot it during the review.

Do we extend any of these models in any way?

Reply:
No, the only references to the stripe polymorphic models provided by `djstripe.stripe_objects` that I could find were in documentation references in `cloudsmith/core/billing/plans.py`